### PR TITLE
Add .gitattributes

### PR DESCRIPTION
**Pull request by lichunqiang**
This  can reduce the size that installed via composer.

### CODE REVIEW

**Comment by boxcla** [comment]
Hi @lichunqiang, thanks for the pull request. Before we can merge it, we need you to sign our Contributor License Agreement. You can do so electronically here: http://opensource.box.com/cla

Once you have signed, just add a comment to this pull request saying, "CLA signed". Thanks!

**Comment by lichunqiang** [comment]
CLA signed

**Comment by boxcla** [comment]
I couldn't find a signed CLA for @lichunqiang. Can you please try again?

**Comment by adrilo** [comment]
@lichunqiang can you educate me on the benefits of having a gitattributes file for this project?

**Comment by lichunqiang** [comment]
@adrilo This mainly can reduce the download files.

http://git-scm.com/docs/gitattributes

Just explode the tests directory when using composer to install this library.

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+/tests              export-ignore
+/.editorconfig      export-ignore
+/.gitattributes     export-ignore
+/.gitignore         export-ignore
+/.scrutinizer.yml   export-ignore
+/.travis.yml        export-ignore
+/composer.lock      export-ignore
+/CONTRIBUTING.md    export-ignore
+/logo.png           export-ignore
+/phpunit.xml   		export-ignore
```
Comment (adrilo) [inline .gitattributes]: indent the last "export-ignore" correctly to be aligned with the other ones

**Comment by lichunqiang** [comment]
Fixed.

**Comment by adrilo** [comment]
Thanks!

**Pull request merged**